packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -259,6 +260,28 @@ abstract contract FundingRateApplier is FeePayer {
         return fundingRate.rate;
     }
 
+    function _validateFundingRate(FixedPoint.Signed memory rate) internal pure {
+        // Constraining range of funding rates limits the PfC for any dishonest proposer and enhances the perpetual's
+        // security. A 1000% funding rate range is equal to a 8.6% PfC for a proposer who can deter honest proposers
+        // for 74 hours: 1000%/year / 360 days / 24 hours * 74 hours max attack time = ~ 8.6%.
+        // Note: 1000% range is equivalent to a funding rate bounds of +500%/-500%. Imagine that the market is very
+        // volatile currently and that the "true" funding rate for the next 74 hours is -500%, but a dishonest
+        // proposer successfully proposes a rate of +500% (after a two hour liveness) and disputes honest
+        // proposers for the next 72 hours. This results in a funding rate error of 1000% for 74 hours, until the DVM
+        // can set the funding rate back to its correct value.
+
+        // Approximate range is corresponding to +/- 500% is [0.000016, -0.000016].
+        FixedPoint.Signed memory MAX_FUNDING_RATE_PER_SECOND =
+            FixedPoint.fromUnscaledInt(500).div(360).div(24).div(60).div(60);
```

Comment:
i wonder if the 500 number should be set as a contract constructor? could this also save cost for the caller if computation was done in constructor and saved.

Reply:
Oh this is a good idea, we could allow the deployer to update this limit?

Reply:
interesting... i think since its a magic number id lean towards yes. But if this value could be shown to be optimal through most use cases id say its ok to hard code.

Reply:
I ended up allowing the ConfigStore to update these max/min values. When we deploy our first Perp we will prob set them closely to the previously hard coded values